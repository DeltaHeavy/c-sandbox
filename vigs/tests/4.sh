#!/bin/bash

echo -n "Test 4: ftable pipe...."
./vigs ham tests/4.in | ./ftable > tests/4.testout
diff tests/4.testout tests/4.out >/dev/null 2>&1
ret=$?
rm tests/4.testout
if [ $ret -ne 0 ]
then
    echo -e "\033[31mFAIL\033[0m"
    false
else 
    echo -e "\033[32mPASS\033[0m"
    true
fi
