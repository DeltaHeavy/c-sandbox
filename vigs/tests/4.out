Total chars: 13
A:         2 ( 15.38%) ****************
B:         0 (  0.00%) 
C:         0 (  0.00%) 
D:         1 (  7.69%) ********
E:         0 (  0.00%) 
F:         1 (  7.69%) ********
G:         0 (  0.00%) 
H:         0 (  0.00%) 
I:         0 (  0.00%) 
J:         0 (  0.00%) 
K:         0 (  0.00%) 
L:         2 ( 15.38%) ****************
M:         0 (  0.00%) 
N:         2 ( 15.38%) ****************
O:         4 ( 30.77%) *******************************
P:         0 (  0.00%) 
Q:         0 (  0.00%) 
R:         0 (  0.00%) 
S:         0 (  0.00%) 
T:         0 (  0.00%) 
U:         1 (  7.69%) ********
V:         0 (  0.00%) 
W:         0 (  0.00%) 
X:         0 (  0.00%) 
Y:         0 (  0.00%) 
Z:         0 (  0.00%) 
